code/modules/mob/living/simple_animal/revenant.dm

```diff
@@ -179,7 +179,7 @@
 	if(sanitize)
 		message = trim(copytext_char(sanitize(message), 1, MAX_MESSAGE_LEN))
 	src.log_talk(message, LOG_SAY)
-	var/rendered = span_revennotice("<b>[src]</b> says, \"[message]\"")
+	var/rendered = span_deadsay(span_command_headset("<b>UNDEAD: [src]</b> says, \"[message]\""))
```

Comment:
still mad about the crossbones.
Related: this seems like something that should be its own span define rather then wrapping existing ones

Reply:
it would only ever be used here

Reply:
that's fine

Reply:
ended up returning the text to normal size anyway